### ISSUE TITLE
Data augmentation

### ISSUE THREAD

**Issue by kevin655** [Alibaba-MIIL/ASL#42]
Hi, thank you for releasing your training code for us to reproduce your models.

I notice the date augmention in your code:
    train_dataset = CocoDetection(data_path_train,
                                  instances_path_train,
                                  transforms.Compose([
                                      transforms.Resize((args.image_size, args.image_size)),
                                      CutoutPIL(cutout_factor=0.5),
                                      RandAugment(),
                                      transforms.ToTensor(),
                                      # normalize,
                                  ]))

Here, RandAugment means the newly proposed augmentation by google? And did you use this augmention except cutout in your models? If so, when reproducing your code, I will use the augmention as you did.

Thanks in advance.

**Comment by mrT23** [comment]
i am not sure i fully understand the question.

- run the code provided, you will reproduce the article results (tested ltresnet on 448)

- for the article i used gpu augmentations, which i cant share (propriety reasons). their main advantage over RandAugment is speed. However, in terms of accuracy, RandAugment, AutoAugment, and the article augmentations are similar. 

- just don't use out-dated HSV augmentations, which are sometimes used in old multi-label repos.

**Comment by kevin655** [comment]
Thank you for your reply about data-augmentation.

That is, in your paper, "Cutout" and "gpu-augmentations" are both used as data augmentations? And the effect of gpu-augmentations and RandAugment are similar in terms of mAP ?

Thanks in advance.

**Comment by mrT23** [comment]
"in your paper, "Cutout" and "gpu-augmentations" are both used as data augmentations"
yes

"the effect of gpu-augmentations and RandAugment are similar in terms of mAP "
yes

**Comment by kevin655** [comment]
OK, got it!

Thanks.
